Common/cpp/NativeModules/NativeReanimatedModule.cpp

```diff
@@ -487,6 +517,28 @@ void NativeReanimatedModule::performOperations() {
     }
   }
 
+  for (const auto &[shadowNode, props] : copiedOperationsQueue) {
+    const jsi::Value &nonAnimatableProps = getNonAnimatableProps(rt, *props);
+    if (nonAnimatableProps.isUndefined()) {
+      continue;
+    }
+    Tag viewTag = shadowNode->getTag();
+    jsi::Value maybeJSPropUpdater =
+        rt.global().getProperty(rt, "updateJSProps");
+    if (maybeJSPropUpdater.isObject()) {
+      jsi::Function jsPropsUpdater =
+          maybeJSPropUpdater.asObject(rt).asFunction(rt);
+      jsPropsUpdater.call(rt, viewTag, nonAnimatableProps);
+#if DEBUG
+    } else {
+      JSLogger jsLogger(jsScheduler_);
+      jsLogger.warnOnJS("[Reanimated] Unable to update JS props.");
+    }
+#else
+    }
```

Comment:
The compiler will optimize away the empty `else` branch here on its own, let's keep things simple.
```suggestion
    } else {
#if DEBUG
      JSLogger jsLogger(jsScheduler_);
      jsLogger.warnOnJS("[Reanimated] Unable to update JS props.");
#endif
    }
```